plugins/themes/default-child

```diff
@@ -0,0 +1 @@
+Subproject commit 5455608894e99d80df20d8f7a1dc38a0d5d84d8b
```

Comment:
Woops, I seem to have included submodule commits for the `bootstrap3` and `default-child` themes. These shouldn't be committed to the main project. Can you revert this before merging @asmecher?